flake.nix

```diff
@@ -57,11 +56,15 @@
             clang = final.clang_20;
             mlir_pkg = final.mlir;
           };
-          llzk-debug = final.callPackage ./nix/llzk.nix {
-            clang = final.clang_20;
-            mlir_pkg = final.mlir-debug;
-            cmakeBuildType = "Debug";
-          };
+          llzk-debug =
+            (final.callPackage ./nix/llzk.nix {
+              clang = final.clang_20;
+              mlir_pkg = final.mlir-debug;
+              cmakeBuildType = "Debug";
+            }).overrideAttrs
+              (attrs: {
+                NIX_CFLAGS_COMPILE = (attrs.NIX_CFLAGS_COMPILE or "") + " -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0";
+              });
```

Comment:
Another way of getting rid of this warning is to set `-O1` in debug builds or not set it at all. I learned that some time after I wrote those flags in the cmake file way back when. In the zirgen I didn't set the optimization level in `Debug` and the warning went away.